web-gateway/app/controllers/UserController.java

```diff
@@ -0,0 +1,79 @@
+package controllers;
+
+import akka.japi.Pair;
+import com.example.auction.bidding.api.*;
+import com.example.auction.item.api.Item;
+import com.example.auction.item.api.ItemData;
+import com.example.auction.item.api.ItemService;
+import com.example.auction.item.api.ItemStatus;
+import com.example.auction.user.api.User;
+import com.example.auction.user.api.UserRegistration;
+import com.example.auction.user.api.UserService;
+import com.lightbend.lagom.javadsl.api.transport.TransportException;
+import org.pcollections.PSequence;
+import play.Configuration;
+import play.data.Form;
+import play.data.FormFactory;
+import play.i18n.MessagesApi;
+import play.mvc.Http;
+import play.mvc.Result;
+import views.html.editItem;
+
+import javax.inject.Inject;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+import static com.example.auction.security.ClientSecurity.authenticate;
```

Comment:
A lot of these `import` statements aren't actually used in the code. Recommend using an IDE function to automatically organize your imports:

- https://www.jetbrains.com/help/idea/optimizing-imports.html
- https://help.eclipse.org/neon/topic/org.eclipse.jdt.doc.user/gettingStarted/qs-OrganizeImports.htm?cp=1_1_0_3_6

You may need to customize the settings for import order to make sure that the imports don't flip-flop back and forth when different people use different IDEs to edit the file.

Here are the settings in my IDE (the defaults for IntelliJ IDEA):

<img width="1438" alt="screen shot 2017-06-22 at 12 01 39 pm" src="https://user-images.githubusercontent.com/44385/27414807-94097222-5742-11e7-99cd-bd44aade9833.png">